src/js/components/CheckBox/stories/CustomToggle.js

```diff
@@ -35,6 +35,9 @@ const customToggleTheme = {
       border: {
         color: undefined,
       },
+      background: {
+        color: 'light-2',
```

Comment:
Putting a default value will break the backward compatibility for users that currently not using background or had the background set in a different way.
the object should be undefined and comment out, and this code should go on the hpe theme.

Reply:
This is added to the custom toggle story ? the object is undefined in the base theme file

Reply:
unless you want me to take this out I totally can just added it to custom story to see an example

Reply:
sorry, my bad. we are good.